git.md

```diff
@@ -143,3 +143,50 @@ code)."
 ```
 
 source: [How to Write a Git Commit Message](https://chris.beams.io/posts/git-commit/#seven-rules)
+
+## Use "change commits" for changes from PR feedback
+
+When a reviewer requests a change during review of a PR, the pull-requester should include special "change commits" to resolve each requested change. The change commit should include a prefix of "Change: " followed by the commit message. 
+
+These change commits will make it simpler to review each change in isolation and make it simpler to determine if a requested change is adequately resolved.
+
+Once all feedback has been resolved, the change commits should be squashed into the relevant commit using `git rebase -i`. It's important to think about which commit your change commit will be squashed with before creating the change commit. This way rebasing is relatively simple.
+
+### Example:
+
+Let's say your feature is a new component, and some configurations need changing to go along with your feature.
+
+```bash
+git commit -m"Add an awesome component to a screen"
+# ...
+git commit -m"Updated some configurations for awesome component"
+```
+
+During review, the reviewer requests a change in your component's code. You then create the change commit.
+
+```bash
+git commit -m"Change: fixed issue with component"
```

Comment:
Git has a built-in convention, where you can do "fixup! Old commit message" as the commit message. Then, if you do a `rebase -i` with the correct flag enabled (don't remember which), it will automatically put this as a fixup commit right after the commit with the message "Old commit message".

Reply:
`git config --global rebase.autosquash true`, then you can do `git commit --squash <offending-commit-hash>`, and git will create the proper commit message for this to happen, so it's all automatic. I think we should embrace this practice in this document somehow.

Reply:
A quick google into what you're talking about and I found this: https://thoughtbot.com/blog/autosquashing-git-commits This appears to reflect what you have mentioned perfectly. Furthermore, I do like the use of fixup over squash in this case (no need to keep the commit's message). I say you're right on here, and I should repurpose this PR to suggest the proper us of "fixup" commits during the review process. Good call!